### DISCUSSION TITLE
plotting not working

### DISCUSSION THREAD

**Discussion by Tanukiman11** [Chia-Network/chia-blockchain#1141] [Q&A]
I have been plotting for about 1 month.  I upgraded to the new node v. 0.2.3 now when I try to plot and view the log it just says loading... and nothing is being written in my temp folder.  Any ideas?  My node is Synced and everything is green... Not sure what to do?

Thank you in advance.

**Comment by hoffmang9** [comment]
That is the issue that caused 0.2.4 to exist. Upgrade for the fix.

**Reply by Tanukiman11** [reply]
Thank you Gene!  You rock.

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
